darwin/exporter/formats/instance_mask.py

```diff
@@ -17,7 +18,7 @@ def export(annotation_files: Generator[dt.AnnotationFile, None, None], output_di
     with open(output_dir / "instance_mask_annotations.csv", "w") as f:
         f.write("image_id,mask_id,class_name\n")
         for annotation_file in get_progress_bar(list(annotation_files), "Processing annotations"):
-            image_id = annotation_file.path.stem
+            image_id = os.path.splitext(annotation_file.filename)[0]
```

Comment:
I have two questions here:

1. If `path.stem` doesn't work, is it because `annotation_file.path` is wrong, or not a path?
2. `os.path` is a bit old, and pretty much anything can now be delegated to `pathlib` https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
Two reasons why I'm using `os.path`:

1. To infer image_id now I'm using filename instead of path, which is a string, so I'd had to convert it to Path anyways
2. If the annotation is part of a video annotation, `filename` has the format `{video_filename}/{frame:07d}.jpg`, so if I use `stem` I would discard the first part, which I need to keep.

So the `pathlib` solution would be: `Path(annotation_file.filename).with_suffix('')`. I thought the `os.path` way looked a bit less convoluted, but I can change it if you like this way better.